src/lib/Sympa/WWW/SOAP.pm

```diff
@@ -1568,6 +1568,99 @@ sub _get_reason_string {
     return $string;
 }
 
+sub moveUser {
+    $log->syslog(
+        'info',
+        '(%s, current_email=%s, email=%s)',
+        @_
+    );
+    my $class         = shift;
+    my $current_email = shift;
+    my $email         = shift;
+
+
+    my $sender                  = $ENV{'USER_EMAIL'};
+    my $robot                   = $ENV{'SYMPA_ROBOT'};
```

Comment:
Starting with the next release (maybe 6.2.73b.1 or 6.2.74), `$ENV{SYMPA_ROBOT}` will be replaced by `$ENV{SYMPA_DOMAIN}` (cf. [this change](https://github.com/sympa-community/sympa/pull/1592/commits/84eadc36f8)).